Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.7-alpine
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /web
+COPY . /web
+
+RUN apk update
+RUN apk add --no-cache --virtual .build-deps \
+    gcc \
+    python3-dev \
+    libffi-dev \
+    libxml2-dev \
+    libxslt-dev \
+    linux-headers \
+    musl-dev \
+    postgresql-dev \
+    ruby \
+    ruby-rdoc \
```

Comment:
We don't need ruby-rdoc.

Reply:
I added it because the container exits with an error about rdoc. I also tried `gem install bundler --no-rdoc` but that didn't work. Let me know if you have any other suggestions.

Reply:
Here is the specific error:
```
ERROR:  While executing gem ... (Gem::DocumentError)
    RDoc is not installed: cannot load such file -- rdoc/rdoc
```

Reply:
That's weird. I'll look into it, thanks! I thought we've removed it from the Ansible configuration.